packages/backend/src/tests/propertyBased/arbitraries/arbitraryLimit.ts

```diff
@@ -0,0 +1,8 @@
+import { fc } from '@fast-check/vitest';
+
+export function arbitraryLimit(): fc.Arbitrary<number> {
+    return fc.integer({
+        min: 1,
+        max: 2147483647,
```

Comment:
Always add a comment when using magic constants, we will forget where we got this from